proposals/lip-0071.md

```diff
@@ -269,7 +272,40 @@ The default reward of the block as a 64-bit unsigned integer.
 
 ### Endpoints for Off-Chain Services
 
-TBD.
+#### getExpectedValidatorRewards
+
+This function is used to estimate the expected rewards for a validator assuming the current BFT weight distribution. It provides an estimate of rewards for one block, one day, one month and one year.
+
+##### Returns
+
+* `reward:` amount of block reward for a validator.
+
+##### Execution
+
+```python
+<<<<<<< HEAD
```

Comment:
Here I had some conflict due to applying a suggestion from github and forgetting to pull it, should be fine now.